aws-api-appsync/src/main/java/com/amplifyframework/core/model/temporal/Temporal.java

```diff
@@ -326,8 +335,7 @@ public Time(@NonNull String text) {
                 zoneOffset = ZoneOffset.from(offsetTime);
             } catch (DateTimeParseException exception) {
                 // Optional timezone offset not present
-                localTime = LocalTime.parse(text, DateTimeFormatter.ISO_LOCAL_TIME);
-                zoneOffset = null;
+                throw new IllegalArgumentException("Failed to create Temporal.Time object from " + text, exception);
```

Comment:
For Temporal.Time, the offset is optional, so this implementation should be very similar to what you did in `Temporal.Date`.  So this method should look something like:

```
try {
    localTime = LocalTime.parse(text, DateTimeFormatter.ISO_LOCAL_TIME);
    zoneOffset = null;
} catch (DateTimeParseException dateTimeParseException) {
    throw new IllegalArgumentException("Failed to create Temporal.Time object from " + text, exception);
}